sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -356,6 +356,10 @@ class TransformWithStateSuite extends StateStoreMetricsTest
       testStream(result, OutputMode.Update())(
         AddData(inputData, "a"),
         CheckNewAnswer(("a", "1")),
+        Execute(q => {
```

Comment:
nit: `{ q =>` This is described in Spark Scala style guide to avoid unnecessary paren.